test/MarketMaker.js

```diff
@@ -196,55 +195,65 @@ contract('marketMaker', function(accounts) {
 
   const periods = 24;
   const tokenTotalSupply = 100;
+  let customerAddressIndex = 1;
 
   var checkScenarioProperties = async function(data, mm, customer) {
-    help.debug("checking scenario", JSON.stringify(data));
+    //help.debug("checking scenario", JSON.stringify(data));
 
-    assert.equal(data.month, await mm.getCurrentPeriodIndex());
+    assert.equal(data.marketMakerMonth, await mm.getCurrentPeriodIndex());
     data.marketMakerEthBalance.should.be.bignumber.equal(web3.eth.getBalance(mm.address));
     data.marketMakerLifBalance.should.be.bignumber.equal(await token.balanceOf(mm.address));
 
     new BigNumber(web3.toWei(tokenTotalSupply, 'ether')).
-      minus(data.burnedTokens).
+      minus(data.marketMakerBurnedTokens).
       should.be.bignumber.equal(await token.totalSupply.call());
-    data.burnedTokens.should.be.bignumber.equal(await mm.totalBurnedTokens.call());
+    data.marketMakerBurnedTokens.should.be.bignumber.equal(await mm.totalBurnedTokens.call());
 
-    if (data.month < periods) {
+    if (data.marketMakerMonth < periods) {
       data.marketMakerBuyPrice.should.be.bignumber.equal(await mm.getBuyPrice());
       assert.equal(data.claimablePercentage, parseInt(await mm.getAccumulatedDistributionPercentage()));
     }
 
-    assert.equal(data.month >= periods, await mm.isFinished());
+    assert.equal(data.marketMakerMonth >= periods, await mm.isFinished());
 
-    data.customerEthBalance.should.be.bignumber.equal(web3.eth.getBalance(customer));
-    data.customerLifBalance.should.be.bignumber.equal(await token.balanceOf(customer));
+    data.ethBalances[customerAddressIndex].should.be.bignumber.equal(web3.eth.getBalance(customer));
+    data.balances[customerAddressIndex].should.be.bignumber.equal(await token.balanceOf(customer));
 
-    data.maxClaimableEth.should.be.bignumber.equal(await mm.getMaxClaimableWeiAmount());
+    data.marketMakerMaxClaimableEth.should.be.bignumber.equal(await mm.getMaxClaimableWeiAmount());
```

Comment:
this shouldnt be `marketMakerMaxClaimableWei` instead of `marketMakerMaxClaimableEth` ? we are comparing it again a wei amount